stock_inventory_valuation_report/reports/stock_inventory_valuation_report.py

```diff
@@ -9,6 +9,9 @@ class StockInventoryValuationView(models.TransientModel):
     _description = 'Stock Inventory Valuation View'
 
     display_name = fields.Char()
```

Comment:
```suggestion

```
If this model has `name` and `reference`, `display_name` is unnecessary.

Reply:
I would keep it in case someone wants to extend the view and use it.
If it's blocking I'll remove

Reply:
IMO, `name` and `reference` can use instead of `display_name`.